mlrun/serving/server.py

```diff
@@ -665,12 +708,42 @@ async def run(body):
     if asyncio.iscoroutine(termination_result):
```

Comment:
The comment is on line 699 - Why we should await each run and not to execute it in parallel? and in the end to use `gather()`?